packages/MonorepoBuilder/src/ComposerJsonDecorator/AutoloadRelativePathComposerJsonDecorator.php

```diff
@@ -149,8 +149,8 @@ private function prefixPath(SplFileInfo $packageComposerFile, $path)
         }
 
         $composerDirectory = dirname($packageComposerFile->getRealPath());
-        $relativeDirectory = Strings::substring($composerDirectory, strlen(rtrim(getcwd(), '/') . '/'));
+        $relativeDirectory = Strings::substring($composerDirectory, strlen(rtrim(getcwd(), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR));
 
-        return $relativeDirectory . DIRECTORY_SEPARATOR . $path;
+        return str_replace(DIRECTORY_SEPARATOR, '/', $relativeDirectory . '/' . $path);
```

Comment:
Could you decouple this method to `normalizeSlashes()`? It could be used on multiple spaces

Reply:
Inside `AutoloadRelativePathComposerJsonDecorator` or another class?

Reply:
Decoupling to standalone class, like `FileSystem` would be better

Reply:
Fixed with 2b0525346649cb5394626819873ea2aca85c13a2